### DISCUSSION TITLE
主页的流量监控数据有问题

### DISCUSSION THREAD

**Discussion by hurriedlu** [1Panel-dev/1Panel#5714] [Q&A]
### 联系方式

hurrkn@163.com

### 1Panel 版本

v1.10.11-lts

### 问题描述

上行流量监控显示不准确，变成了 下行*2+上行 的数值。

![image](https://github.com/1Panel-dev/1Panel/assets/84308195/a000ec09-284f-49a7-98da-5da6baef3abc)

### 重现步骤

不需要任何步骤，显示的一直有问题

### 附加信息

运行环境或者说是访问路径：1panel主机本身不允许外网访问，通过同一网络内的另一台主机的nginx反向代理实现访问1panel主机

**Comment by wanghe-fit2cloud** [comment]
感谢反馈，我们先在本地环境复现一下上述问题。

**Comment by ssongliu** [comment]
这里的上行和下行并没有直接的关系 ，可以尝试往服务器上传个大文件，然后观察一下数值变化
